cmake/Reprojucer.cmake

```diff
@@ -51,7 +53,12 @@ function(jucer_project_begin project_name)
         )
       endif()
 
-      if(tag STREQUAL "PROJECT_VERSION")
+      if(POLICY CMP0054)
```

Comment:
No. `if(POLICY X)` is used to check that the current version of CMake knows about that policy. For instance, CMP0054 was introduced in CMake 3.1 and `cmake_policy(SET CMP0054 NEW)` would warn or fail on older versions.